sys-cluster/kube-router/kube-router-0.4.0.ebuild

```diff
@@ -0,0 +1,160 @@
+# Copyright 2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+EGO_PN="github.com/cloudnativelabs/${PN}"
+
+inherit golang-build golang-vcs-snapshot
+
+KEYWORDS="~amd64"
+
+DESCRIPTION="A turnkey solution for Kubernetes networking"
+HOMEPAGE="https://kube-router.io"
+SRC_URI="https://github.com/cloudnativelabs/kube-router/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="Apache-2.0 MIT BSD BSD-2 MPL-2.0 ISC LGPL-3-with-linking-exception"
+
+# It will fail (timeout) at least with standard set of FEATURES attempting to serve bgp
+RESTRICT="test"
+
+SLOT="0"
+
+RDEPEND="
+	net-firewall/iptables[conntrack]
+	net-firewall/ipset
+	sys-cluster/ipvsadm
+"
+
+# Vendored dependencies:
```

Comment:
I don't think this list is needed.

Reply:
It helps with tracking licenses, see bug https://bugs.gentoo.org/694792
I'll add a comment

Reply:
done